.github/workflow_scripts/env_setup.sh

```diff
@@ -12,7 +12,8 @@ function setup_build_env {
 function setup_build_contrib_env {
     python3 -m pip install --upgrade pip
     python3 -m pip install -r $(dirname "$0")/../../docs/requirements_doc.txt
-    python3 -m pip install git+https://github.com/zhanghang1989/d2l-book
+    # removing this package temporarily to avoid build docs failure
+    # python3 -m pip install git+https://github.com/zhanghang1989/d2l-book
```

Comment:
Why not completely drop this d2lbook's fork setup? AFAICS, it's not used in the docs. AutoGluon moved away from d2lbook to furo sometime ago with https://github.com/autogluon/autogluon/pull/2924.

Ps this issue was reported earlier on an older version of d2lbook: https://github.com/d2l-ai/d2l-book/issues/75

```suggestion
```

Reply:
Thanks Anirudh, I came across this issue in d2l but it is still an open issue there.
I think it is not used anymore as well, but plan to remove it once I consult with the team today, so I have just commented it for now

Reply:
Its open since I couldn't reproduce the bug in the latest release of `d2lbook` and even here the forked installation is probably based of a very old version of the repo.